### PR TITLE
429 implement recent dashboard layout changes to text tab

### PR DESCRIPTION
**Pull request by laurasootes**
I added/translated the changes made to the text page of the dashboard (update and stop button, selection of model, parameters of model).
Everything seems to work, except for the issues that I created (#435 and #437 ), which are not problems in the translation but merely problems that already existed in the text page that we did not know.

This branches also fixes the linting error about the tensor flow version

### CODE REVIEW

**Comment by cpranav93** [comment]
Hey @laurasootes, could you provide me with the model for testing the text explanations? I currently only have the onnx model for the images. Thanks!

**Comment by laurasootes** [comment]
The model is already in the repo under app_data. It is the movie review model.

**Comment by cpranav93** [comment]
Hey @laurasootes, I am trying out the dashboard>text and no matter what text I give, I get an error (see attached screenshot). I am not sure if I am doing something wrong here.

image

**Comment by laurasootes** [comment]
but can you find where and why it is going wrong in the code?

**Comment by cpranav93** [comment]
I had a look at where the error was happening and it is in line 546: 
hti = Html2Image()

I do not know enough about dash to figure out yet why this is causing an exception which gives the error message.

Also, I noticed that when I refresh the page, and input just the text and model without the method, it raises
'There was an error running the model. Check either the test text or the model.' 
but it should raise:
'Missing model, input text or XAI method.'

This can be fixed in line 489 by changing it to:
if (fn_m and input_text and sel_methods) is not None:

File: dashboard/layouts.py [hunk 1]
```diff
@@ -650,29 +785,29 @@ def get_uploads_text():
...
-                    id='graph_text_rise',
+                    id='graph_text_lime',
                     figure = utilities.blank_fig())],
                     className = 'row',
                     style = {
                         'margin-top': '80',
                         'margin-left': '140px',
                         'height': '100px'
-                    })
+                    })
```
Comment (loostrum) [inline dashboard/layouts.py:810]: The extra blank space at the end is not needed here :)

**Review by loostrum — APPROVED** [review]

**Pull request merged**